.circleci/config.yml

```diff
@@ -15,7 +15,6 @@ executors:
       - image: circleci/ruby:2.7.3-node-browsers
         environment:
           RAILS_ENV: test
-          BUNDLER_VERSION: 1.17.3
```

Comment:
do we need to set this to a specific version to match Gemfile.lock?

Reply:
Build throws a warning, so I'll re-add the install with the changed version

```
Warning: the running version of Bundler (2.1.4) is older than the version that created the
lockfile (2.2.20). We suggest you to upgrade to the version that created the lockfile by
running `gem install bundler:2.2.20`.
```